### PR TITLE
add v2 stop order endpoints and v2 replace order endpoint

### PR DESCRIPTION
**Pull request by EricZ11**

### CODE REVIEW

**Comment by dextertd** [comment]
New Set Leverage endpoint is the same as the abandoned one? `/user/leverage/save`

File: locales/en.yml [hunk 1]
```diff
@@ -128,6 +128,8 @@ en:
   row_comment_from: |
     From ID. Default: return latest data
   row_comment_limit: Limit for data size per page, max size is 50. Default as showing 20 pieces of data per page
+  row_comment_cursor: Page turning mark，Use return <code>cursor</code>
```
Comment (dextertd) [inline locales/en.yml]: How does `row_comment_cursor` work? I don't understand.

Comment (EricZ11) [reply locales/en.yml]: it's similar `/v2/private/exchange-order/list`  ->  from

Comment (dextertd) [reply locales/en.yml]: I still don't really understand... Could you provide sample data that would be sent as the value of `cursor`?

**Review by dextertd — CHANGES_REQUESTED** [review]
Review PR #121

File: locales/en.yml (hunk 1 above)
Comment (EricZ11) [reply locales/en.yml]: urlencode after: GMts5r8FUBO2QWB3gqsbjqD%2BEM0dSlVjbxtxDTs%2Bns3x29g%2BTvMtbRAa

**Pull request merged**